tests/app/main/views/test_dashboard.py

```diff
@@ -833,7 +833,7 @@ def test_should_show_upcoming_jobs_on_dashboard(
         'main.service_dashboard',
         service_id=SERVICE_ONE_ID,
     )
-    mock_get_jobs.assert_called_once_with(SERVICE_ONE_ID)
+
     mock_get_scheduled_job_stats.assert_called_once_with(SERVICE_ONE_ID)
```

Comment:
This doesn't get called anymore after demo changes to the uploads/letter item in sidebar. Dashboard is correctly showing upcoming jobs, but we might want to write a test for this function, since it's not called at all anymore in the current dashboard tests

Reply:
Agreed on writing a test. Can you either add that in this PR or create a new issue to make sure it's not forgotten?

Reply:
Removed this patch from the fixtures list for a few tests where it is (correctly) not being used, and added `assert_called_once` for this patch into a few additional tests to validate that it is (correctly) being called while providing various dashboard features.